### ISSUE TITLE
can add a vpn, also says check script in not working as well as iptables

### ISSUE THREAD

**Issue by tigomonkey** [alpha-si/vpnman#3]
also says check script in not working as well as iptables. im on ubuntu and idk whats up

**Comment by federico-alphasi** [comment]
Can you provide more info?
which error do you have when add a vpn?
The fail of iptables check may be related to missing/incorrect installation steps. Have you exceuted correctly all the steps reported at https://github.com/alpha-si/vpnman/wiki/Installation#install-frontend ?

**Comment by tigomonkey** [comment]
Yea all the commands were there just the errors: check script error
[error1]  and iptables check one [error1]. I was on Ubuntu 16.04 Lts, do I
download openvpn  and easy-rsa? When ever I tried to make a vpn it would
say also cannot find file template/srvcnfg.tmpl  or something along those
lines

**Comment by federico-alphasi** [comment]
Yes, openvpn must be installed on your system. The template/srvcfg.tmpl error may be due to missing path. Have you correctly set the VPN_ROOT_PATH in the "Edit Configuration" form? you must set the absolute path to the vpnman's vpn folder (for example, if you have installed vpnman under "/var/www/html/", VPN_ROOT_PATH should be set to "/var/www/html/vpnman/vpn/")

**Comment by tigomonkey** [comment]
figured that out, do i leave /etc/openvpn blank then?  it just has
update-resolv-conf in it i think.

**Comment by tigomonkey** [comment]
also you can check it out i cant get a vpn to run the url is freedommail.ga

**Comment by federico-alphasi** [comment]
It seems that ovpnctrl daemon is missing. Have you succesfully compiled the ovpnctrl daemon?
Instructions are here: https://github.com/alpha-si/vpnman/wiki/Installation#compile-ovpnctrl-daemon
After "make install" the ovpnctrl binary is automatically copied under vpn/bin/
The vpn page doesn't show your vpn due to a bug when ovpnctrl daemon is not correctly installed.
I plan to fix it as soon as possible.

**Comment by tigomonkey** [comment]
yea everything compiled accordingly, i mean i followed everything to the  T
there were no error messages when compiling the daemon.

**Comment by federico-alphasi** [comment]
Have you used the correctly apache user with visudo? the installation instructions use "apache" but some OS may use a different user (for example "www-data")

**Comment by tigomonkey** [comment]
Ya i use apache on ubuntu its user is www-data. So in visudo i add www-data
to sudoers. Im curious is all the vpn config files and certs stored in the
web root directory? And then openvpn just gets its directives from there?

**Comment by federico-alphasi** [comment]
Vpn configs and certs are create and stored in the vpnman's vpn folder. Openvpn server is started using these configuration files.  Openvpn server up-script is used to start the ovpnctrl daemon, it's a bridge between openvpn server and vpnman web platform. In your case there are some troubles starting this daemon. You can try to manually start the openvpn server from terminal (from vpnman's vpn folder): 
./bin/openvpn.sh start ./vpnXX/vpnXX_server.conf    (XX is your vpn id)
